tests/performance/nodedensity/nodedensity_test.go

```diff
@@ -69,7 +69,7 @@ var _ = Describe("Application deployment test in Perfronace test EdgeNodes", fun
 			podlist = metav1.PodList{}
 			runtime := b.Time("runtime", func() {
 				NoOfEdgeNodes = 10
-				podlist = HandleEdgeDeployment(cloudHub, ctx.Cfg.ApiServer2+DeploymentHandler, ctx.Cfg.ApiServer2+NodeHandler,
+				podlist = HandleEdgeDeployment(cloudHub, ctx.Cfg.ApiServer2+DeploymentHandler, ctx.Cfg.ApiServer+NodeHandler,
```

Comment:
what is ApiServer2 ? is it the api server where the cloud is running ?
can we rename to something meaningul ? one  apiserver is for cloud and other is for edge nodes right ?

Reply:
Okay. I will change all of the configs about these